CoreTests/Integration/Contacts/Create.cs

```diff
@@ -75,5 +76,49 @@ public void create_complex_contact()
             Assert.AreEqual("John", contact.ContactPersons[0].FirstName);
             Assert.AreEqual(expectedAccountNumber, contact.AccountNumber);
         }
+
+        [Test]
+        public void create_contact_with_tracking()
+        {
+            //Test will create a Tracking Categorie and use it if there are none available.
+            Api.Contacts.SummarizeErrors(true);
```

Comment:
@philals Does summarize errors give any benefit here? Or is this added following the misunderstanding of it that we discussed the other day?